Makefile

```diff
@@ -33,12 +34,14 @@ ifeq ($(WEB_UI),true)
 _LORAX_ARGS += -i anaconda-webui
 endif
 
-# Step 7: Buid end ISO
+# Step 7: Build end ISO
 ## Default action
 build/deploy.iso:  boot.iso container/$(IMAGE_NAME)-$(IMAGE_TAG) xorriso/input.txt
 	mkdir $(_BASE_DIR)/build || true
 	xorriso -dialog on < $(_BASE_DIR)/xorriso/input.txt
 	implantisomd5 build/deploy.iso
+	mv build/deploy.iso build/$(ISO_NAME).iso
```

Comment:
If we are supporting renaming as part of the action, maybe it should be included as a step in the action instead? I was having issues doing it that way without doing it in the makefile.

Reply:
Renaming isn't part of the action. Renaming is just done for the test artifact so it's easier to identify what it built. The action always outputs `deploy.iso`. Previously to the `${{ github.workspace }}/build`, but with the extra input it would be whichever directory is specified. The consistent name makes using `make` easier and the Makefile easier to understand. If the file needs to be renamed, it can be done outside of the action as it is done in the test.

Reply:
Ok, so your expectation is the user of the action must rename the file in a separate process themselves? The benefit of what I've done here is they can define the iso_name as an input and the rename process doesn't need to be created by the user of the action, it's just done as part of it.

Reply:
At least move it to the action instead of in the `Makefile`. You can embed it as a step after the `docker run` step. That way the result of `make` is consistent and the `make` command is consistent while still embedding the rename in the action.

Reply:
Sounds good! I can move it to the action.